users/api/http/transport.go

```diff
@@ -114,6 +182,8 @@ func encodeError(_ context.Context, err error, w http.ResponseWriter) {
 		w.WriteHeader(http.StatusBadRequest)
 	case users.ErrUnauthorizedAccess:
 		w.WriteHeader(http.StatusForbidden)
+	case users.ErrUserNotFound:
+		w.WriteHeader(http.StatusBadRequest)
```

Comment:
Change error name, or send different status code. For not found it makes sense to return `404`.